CMakeLists.txt

```diff
@@ -173,7 +173,7 @@ if (USE_CRYPTO)
 endif ()
 
 if(KVZ_BUILD_SHARED_LIBS)
-  list( APPEND CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib" "./" "../lib" )  
+  list( APPEND CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}" "./" "../lib" )  
```

Comment:
Great, but `CMAKE_INSTALL_LIBDIR` can be an absolute path on some platforms like NixOS... Perhaps just use `${CMAKE_INSTALL_FULL_LIBDIR}` instead of `${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}`?

Reply:
I remember that problem on openSUSE, years ago. I thought it had been solved. Not everywhere, it comes out.